improver/wxcode/weather_symbols.py

```diff
@@ -452,13 +452,23 @@ def create_symbol_cube(cubes):
             if coord.name() in ["forecast_period", "time"]:
                 coord.bounds = None
 
-        attributes = generate_mandatory_attributes(cubes)
+        mandatory_attributes = generate_mandatory_attributes(cubes)
+        optional_attributes = weather_code_attributes()
+        model_configurations = [
+            c.attributes["mosg__model_configuration"]
+            for c in cubes
+            if c.attributes.get("mosg__model_configuration")
+        ]
+        if model_configurations:
+            optional_attributes["mosg__model_configuration"] = " ".join(
+                sorted(set(model_configurations))
+            )
```

Comment:
I think this is in the wrong place. `mosg__` indicates that this is a an attribute specific to Met Office data. Searching through our code base, nearly all references to `mosg__` are restricted to `metadata/constants/mo_attributes.py` and functions in `metadata/amend.py`. I think these lines of code belong in a new method in `amend.py`.
I also think these lines should be modified so that ALL or NONE of the cubes have to have the attribute, as it stands, it allows a mixture which would indicate something rather dodgy going on.

Reply:
This code has now been moved into `metadata/amend.py` as suggested.